ipc/cli/src/commands/checkpoint/relayer.rs

```diff
@@ -74,6 +80,12 @@ pub(crate) struct BottomUpRelayerArgs {
     pub subnet: String,
     #[arg(long, short, help = "The number of seconds to submit checkpoint")]
     pub checkpoint_interval_sec: Option<u64>,
+    #[arg(
```

Comment:
This can generally be `1` or `2` as in Tendermint we don´t  expect re-orgs. That being said, this may come handy once we support other types of consensus algorithms in child subnets.

Reply:
Currently it is default to 0 if user does not set it.